eng/pipelines/release-go-start-pipeline.yml

```diff
@@ -10,6 +10,12 @@ parameters:
       space at the beginning of each line.
     type: object
 
+  - name: isSecurityRelease
+    displayName: >
+      This release includes security fixes:
+    type: boolean
```

Comment:
I think you reverted the string change here, I only meant to change the go-images pipeline. Please change this one back to True, False and Cancel with Cancel as default

Reply:
done